powerauth-java-crypto/src/main/java/io/getlime/security/powerauth/crypto/lib/encryptor/EncryptorFactory.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * PowerAuth Crypto Library
+ * Copyright 2023 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getlime.security.powerauth.crypto.lib.encryptor;
+
+import io.getlime.security.powerauth.crypto.lib.encryptor.ecies.ClientEciesEncryptor;
+import io.getlime.security.powerauth.crypto.lib.encryptor.ecies.EciesRequestResponseValidator;
+import io.getlime.security.powerauth.crypto.lib.encryptor.ecies.ServerEciesEncryptor;
+import io.getlime.security.powerauth.crypto.lib.encryptor.exception.EncryptorException;
+import io.getlime.security.powerauth.crypto.lib.encryptor.model.EncryptorId;
+import io.getlime.security.powerauth.crypto.lib.encryptor.model.EncryptorScope;
+import io.getlime.security.powerauth.crypto.lib.encryptor.model.EncryptorSecrets;
+import io.getlime.security.powerauth.crypto.lib.encryptor.model.EncryptorParameters;
+
+/**
+ * The {@code EncryptorFactory} class provide high level encryptors for PowerAuth End-To-End encryption implementation.
+ * You can construct both server and client-side constructors in this factory.
+ */
+public class EncryptorFactory {
+    /**
+     * Default constructor.
+     */
+    public EncryptorFactory() {}
```

Comment:
Nitpick: It is not default, but no-arg constructor and is not needed.

Reply:
Removed